### PR TITLE
Add feature flag info to READMEs

### PR DESCRIPTION
**Pull request by camigthompson**
Addresses: https://github.com/NuGet/Engineering/issues/4621

### CODE REVIEW

File: src/NuGet.Jobs.Db2AzureSearch/README.md [hunk 1]
```diff
@@ -65,6 +65,7 @@ Once you've created your Azure resources, you can create your `settings.json` fi
 * The `SearchServiceApiKey` setting is an admin key that has write permissions to the Azure Search resource. Make sure the Azure Search resource you're connecting to has API keys enabled (either in parallel with managed identities "RBAC" access or with managed identities authentication disabled).
 * The `StorageConnectionString` and `AuxiliaryDataStorageConnectionString` settings are both the connection string to your Azure Blob Storage account.
 * The `DownloadsV1JsonUrl` setting is the URL to `downloads.v1.json` file above. Make sure it works without authentication.
+* The `FeatureFlags:ConnectionString` setting is the connection string to your Azure Blob storage account.
```
Comment (agr) [inline src/NuGet.Jobs.Db2AzureSearch/README.md:68]: Could you please clarify the purpose of the blob storage account this connection string points to?

Comment (camigthompson) [reply src/NuGet.Jobs.Db2AzureSearch/README.md:68]: Should it be added here or earlier in the document when it discusses the resources needed to run this job? (lines 47-51)

Comment (agr) [reply src/NuGet.Jobs.Db2AzureSearch/README.md:71]: Doesn't really matter as long as there is clarity about the purpose of that connection string. There are now 3 connection strings and it is not very clear why.

There is a description of expected blob storage content at line 53, continuing with expected location of a feature flags file would be good.

File: src/NuGet.Jobs.Db2AzureSearch/README.md [hunk 2]
```diff
@@ -54,6 +54,9 @@ In your Azure Blob Storage account, you will need to create a container named `n
 1. `downloads.v1.json` with content `[]`
 1. `ExcludedPackages.v1.json` with content `[]`
 
+You will also need to create a second container (if it does not already exist) named `content` and upload the following file:
+1. `flags.json ` with content `{}`
```
Comment (agr) [inline src/NuGet.Jobs.Db2AzureSearch/README.md:58]:
```suggestion
1. `flags.json` with content `{}`
```

**Review by drewgillies — APPROVED** [review]

**Review by joelverhagen — APPROVED** [review]

**Pull request merged**